pandas/core/dtypes/inference.py

```diff
@@ -398,8 +398,11 @@ def is_dict_like(obj):
     >>> is_dict_like([1, 2, 3])
     False
     """
+    for attr in ("__getitem__", "keys", "__contains__"):
```

Comment:
not a big deal, but can you expand the is_dict_like test to account for this?

Reply:
Sure thing.  I’ll do that and merge afterwards.

Reply:
Sure thing.  I’ll do that and merge afterwards.

Reply:
Sure thing.  I’ll do that and merge afterwards.